ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -34,6 +34,8 @@ namespace ArchiSteamFarm.IPC {
 	internal static class ArchiKestrel {
 		private const string ConfigurationFile = nameof(IPC) + ".config";
 
+		internal static string WebsiteDirectory;
```

Comment:
Prive write modifier + init to default dir and eventual correction to custom in `if`.